### DISCUSSION TITLE
OpenAPIRuntime and OpenAPIURLSession

### DISCUSSION THREAD

**Discussion by bspringston** [skiptools/skip#39] [Ideas]
Apple recently shipped the ability to generate an api client based on an openapi.yaml file.  Any thoughts/plans to support this in the near future? 
https://developer.apple.com/wwdc23/10171

**Comment by aabewhite** [comment]
It would certainly be interesting to see if Skip could just run on the Swift output and translate it into Kotlin. We'll add this as something we need to investigate, but I can't make any promises on a timetable right now. Please let us know the results if you do any of your own investigation, and thanks for pointing this out!
